json-modules/pom.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>json-modules</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
no need to define version number explicitly if it is the same as parent's.
Merging this PR for now, please raise a delta PR with this version removed.